gvr-videoplayer/app/src/main/java/org/gearvrf/videoplayer/provider/asyntask/AlbumAsyncTask.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright 2015 Samsung Electronics Co., LTD
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.gearvrf.videoplayer.provider.asyntask;
+
+import android.content.Context;
+import android.database.Cursor;
+import android.database.CursorWrapper;
+import android.os.AsyncTask;
+import android.provider.MediaStore.Video;
+import android.provider.MediaStore.Video.VideoColumns;
+import android.support.annotation.NonNull;
+import android.util.Log;
+
+import org.gearvrf.videoplayer.VideoPlayerApp;
+import org.gearvrf.videoplayer.model.Album;
+
+import java.util.LinkedList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class AlbumAsyncTask extends AsyncTask<Void, Void, List<Album>> {
+
+    private static final String TAG = AlbumAsyncTask.class.getSimpleName();
+    private GetDataCallback<List<Album>> mGetDataCallback;
+
+    public AlbumAsyncTask() {
+    }
+
+    public AlbumAsyncTask(@NonNull GetDataCallback<List<Album>> mGetDataCallback) {
+        this.mGetDataCallback = mGetDataCallback;
+    }
+
+    @Override
+    protected List<Album> doInBackground(Void... voids) {
+        return load();
+    }
+
+    public List<Album> load() {
+
+        Context context = VideoPlayerApp.getInstance().getApplicationContext();
+        List<Album> albums = new LinkedList<>();
+
+        String[] projection = new String[]{
+                VideoColumns._ID,
+                VideoColumns.BUCKET_DISPLAY_NAME,
+                VideoColumns.TITLE,
+                VideoColumns.DATA,
+                VideoColumns.DURATION,
+                VideoColumns.WIDTH,
+                VideoColumns.HEIGHT,
+                "is_360_video"
```

Comment:
Where does is_360_video come from? On Oculus GO there are frequent crashes because this column doesn't exist.

Reply:
Samsung has its own media provider tables with some more fields besides the default of android.

Reply:
Oculus GO doesn't have these so the app needs to be able to tolerate it.

Reply:
It can be removed from the app. The application has some other methods to detect 360 videos.